.github/workflows/rust.yml

```diff
@@ -100,16 +100,22 @@ jobs:
       if: ${{ matrix.test && matrix.os != 'windows-latest' }}
       run: |
         set -euxo pipefail
-        for bin in $bins; do ./${{ matrix.output }} binstall --no-confirm $bin; done
-        ./${{ matrix.output }} binstall --manifest-path . --no-confirm cargo-binstall
+        for bin in $bins; do ./${{ matrix.output }} binstall --log-level debug --no-confirm $bin; done
+        for bin in $test_bins; do $HOME/.cargo/bin/$bin --version; done
+        cargo binstall --help >/dev/null
+        ./${{ matrix.output }} binstall --log-level debug --manifest-path . --no-confirm cargo-binstall
+        cargo binstall --help >/dev/null
       env:
         bins: cargo-deb cargo-llvm-cov cargo-binstall
+        test_bins: cargo-deb cargo-llvm-cov
 
     - name: Test (Windows)
       if: ${{ matrix.os == 'windows-latest' }}
       run: |
         ./${{ matrix.output }} binstall --no-confirm cargo-binstall
+        cargo binstall --help
         ./${{ matrix.output }} binstall --manifest-path . --no-confirm cargo-binstall
+        cargo binstall --help
```

Comment:
This is to test that the binary can run?

Reply:
Yes, it is testing that the binary installed can be run.